rero_ils/translations/messages.pot

```diff
@@ -2477,98 +2477,58 @@ msgstr ""
 msgid "Software under development"
 msgstr ""
 
-#: rero_ils/templates/rero_ils/frontpage.html:78
```

Comment:
This does not come from your PR. It seems that translations are not up to date. Can you check that we do not loose translations from transifex. You have similar cases latter in this file.